modules/nw-network-config.adoc

```diff
@@ -25,6 +25,11 @@ Phase 1:: After entering the `openshift-install create install-config` command.
 +
 For more information on these fields, refer to "Installation configuration parameters".
 
+[NOTE]
+====
+Set `networking.machineNetwork` to match the CIDR which the preferred NIC resides.
```

Comment:
Suggest: 

s/`networking.machineNetwork`/the `networking.machineNetwork` field/